equinix/provider.go

```diff
@@ -249,51 +248,3 @@ func isEmpty(v interface{}) bool {
 		return false
 	}
 }
-
-func slicesMatch(s1, s2 []string) bool {
-	if len(s1) != len(s2) {
-		return false
-	}
-	visited := make([]bool, len(s1))
-	for i := 0; i < len(s1); i++ {
-		found := false
-		for j := 0; j < len(s2); j++ {
-			if visited[j] {
-				continue
-			}
-			if s1[i] == s2[j] {
-				visited[j] = true
-				found = true
-				break
-			}
-		}
-		if !found {
-			return false
-		}
-	}
-	return true
-}
-
-func slicesMatchCaseInsensitive(s1, s2 []string) bool {
```

Comment:
This was used in tests and in one resource, so it has been duplicated to facilitate splitting the tests apart.  While we could replace it with newer functions from an external package, the resource that uses it is ECX and will disappear in a few months, and the tests that use it would be better updated by the owning team.  Duplicating the function puts us in a better position for future refactorings than aiming for perfection here.